dumpling/tests/basic/run.sh

```diff
@@ -88,6 +88,12 @@ actual=$(sed -n '2p' ${DUMPLING_OUTPUT_DIR}/result.000000000.csv)
 echo "expected 2, actual ${actual}"
 [ "$actual" = 2 ]
 
+# Test for dump with sequence
+run_dumpling | tee ${DUMPLING_OUTPUT_DIR}/dumpling.log
+actual=$(grep -w "dump data successfully" ${DUMPLING_OUTPUT_DIR}/dumpling.log|wc -l)
```

Comment:
This can't prove this dump task succeeds. You need to truncate log first or grep failure dumpling.log and make sure it's 0.